README.md

```diff
@@ -274,18 +283,36 @@ Some prefixes can be combined with numbers, which limit them to matching specifi
 | `etcetra/list/nickname`                | `SPECIAL:`   |                                                                   |
 | `list/random_name`                     | `NAME[x]:`   | `x` is the column (1-2 surname, 3-4 female given, 5-6 male given) |
 
-
-
-
 ## Tools
 
 [//]: # "- [TranslationSync](https://github.com/IllusionMods/TranslationTools#translationsync) is a plugin for formatting and copy/pasting translations between files when there are duplicate entries."
-- [Release Tool](https://github.com/SpockBauru/TranslationTools_Illusion#releasetool) - Tool that cleans up the translation files to remove any unnecessary untranslated parts (in this repo).
+- [Release Tool](https://github.com/SpockBauru/TranslationTools_Illusion#releasetool) - Tool that cleans up the translation files to remove any unnecessary untranslated parts.
 - [Yomichan](https://foosoft.net/projects/yomichan/) - This browser plugin allows you to see the definition of Japanese words by putting your mouse over them in the browser and pressing shift.
 - Dictionaries:
   - https://jisho.org/
   - http://www.romajidesu.com/
 
+
 ## Status
 
-Right now the project is at a fast pace and things are constantly changing. Coordinate with other translators on the [Koikatsu Discord](https://discord.gg/urDt8CK) #lingual-studies channel. To avoid translation conflicts please ask if anyone is working on a file. If you have any questions about the quality of your translations, ask for advice on the server.
+Right now 37% of the game is translated and we are prioritizing the UI and game items. The project is evolving at a fast pace and things are constantly changing, so coordinate with other translators in the [Koikatsu Discord](https://discord.gg/urDt8CK) #lingual-studies channel. To avoid translation conflicts please take a look in the To Do list on #whiteboard channel and ask if anyone is working on a file in #lingual-studies channel. If you have any questions about the quality of your translations, ask for advice on the server.
+
+### To Do List
+
+**Translate Tutorials:** Delete `KoikatsuSunshine\UserData\save\Global.dat` to make the tutorials appear again (make a backup before!)
+
+- <s>Translate Chara Maker Tutorial</s> Taken by xmex
```

Comment:
```suggestion
- ~~Translate` Chara Maker Tutorial~~ Taken by xmex
```

Reply:
Double tilde is not working in the preview, even if it appear in the "edit" window. I also tried a handful of Strikethrough commands and the only one that worked was the <s></s>

Reply:
That's odd, it's in the github documentation, but ok.